### PR TITLE
MINOR: Some small cleanups/improvements to KAFKA-5031

### PR DESCRIPTION
**Pull request by hachikuji**

### CODE REVIEW

**Review by ijuma — APPROVED** [review]
Thanks for the PR, LGTM.

**Comment by hachikuji** [comment]
Thanks. I'll merge after the builds complete.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk8-scala2.12/5421/
Test PASSed (JDK 8 and Scala 2.12).

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk7-scala2.11/5436/
Test PASSed (JDK 7 and Scala 2.11).

**Comment by gosubpl** [comment]
@ijuma @hachikuji many thanks for the effort you put in reviewing https://github.com/apache/kafka/pull/3156 . I learned a lot from that discussion and also from this PR. It takes a while to get adapted to idiomatic style of a particular project :)

**Comment by ijuma** [comment]
Thanks for your contribution!

**Comment by hachikuji** [comment]
@gosubpl Yes, thanks a lot for the patch! I thought we might have to let the issue slip, so I was glad you picked it up and we were able to get it into the release.

File: clients/src/test/java/org/apache/kafka/common/record/DefaultRecordTest.java
Comment (ijuma) [inline clients/src/test/java/org/apache/kafka/common/record/DefaultRecordTest.java]: Typo in the comment, should be `value` instead of `key`.

**Review by ijuma — APPROVED** [review]
LGTM, just one typo to be fixed before merging.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk8-scala2.12/5436/
Test PASSed (JDK 8 and Scala 2.12).

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk7-scala2.11/5451/
Test FAILed (JDK 7 and Scala 2.11).

**Comment by ijuma** [comment]
Merged to trunk and 0.11.0.

**Pull request closed without merging**
